app/views/Moderation/CourierPackages/Actions/index.tsx

```diff
@@ -73,6 +79,16 @@ function Actions(props: Props) {
             >
                 Books
             </Button>
+            <Button
+                className={styles.button}
+                name={undefined}
+                title="View school packages"
+                onClick={showSchoolPackagesModal}
+                disabled={disabled}
+                variant="tertiary"
+            >
+                Schools
+            </Button>
```

Comment:
it's better to show this button conditionally. If `schoolPackages` exits in given `courierPackage`, then show this button.